addons/l10n_account_edi_ubl_cii_tests/tests/test_xml_cii_fr.py

```diff
@@ -0,0 +1,272 @@
+            'amount': 10,
+            'type_tax_use': 'sale',
+            'country_id': cls.env.ref('base.fr').id,
+        })
+
+        cls.tax_0 = cls.env['account.tax'].create({
+            'name': 'tax_0',
+            'amount_type': 'percent',
+            'amount': 0,
+            'type_tax_use': 'sale',
+            'country_id': cls.env.ref('base.fr').id,
+        })
+
+        cls.acc_bank = cls.env['res.partner.bank'].create({
+            'acc_number': 'FR15001559627231',
+            'partner_id': cls.company_data['company'].partner_id.id,
+        })
+
+        cls.invoice = cls.env['account.move'].create({
+            'move_type': 'out_invoice',
+            'journal_id': cls.journal.id,
+            'partner_id': cls.partner_1.id,
+            'partner_bank_id': cls.acc_bank,
+            'invoice_date': '2017-01-01',
+            'date': '2017-01-01',
+            'currency_id': cls.currency_data['currency'].id,
+            'invoice_line_ids': [(0, 0, {
+                'product_id': cls.product_a.id,
+                'product_uom_id': cls.env.ref('uom.product_uom_dozen').id,
+                'price_unit': 275.0,
+                'quantity': 5,
+                'discount': 20.0,
+                'tax_ids': [(6, 0, cls.tax_21.ids)],
+            })],
+        })
+    @classmethod
+    def setup_company_data(cls, company_name, chart_template):
+        # OVERRIDE
+        # to force the company to be french
+        res = super().setup_company_data(
+            company_name,
+            chart_template=chart_template,
+            country_id=cls.env.ref("base.fr").id,
+            phone='+1 (650) 555-0111',  # [BR-DE-6] "Seller contact telephone number" (BT-42) is required
+            email="info@yourcompany.com",  # [BR-DE-7] The element "Seller contact email address" (BT-43) is required
+        )
+        return res
+
+    ####################################################
+    # Test export - import
+    ####################################################
+
+    def test_export_pdf(self):
+        self.invoice.action_post()
+        pdf_values = self.edi_format._get_embedding_to_invoice_pdf_values(self.invoice)
+        self.assertEqual(pdf_values['name'], 'factur-x.xml')  # first, make sure the old factur-x is not installed !
+
+        # create a new invoice -> generates all the xmls (if multiple), as if we created an invoice in the UI.
+        invoice = self._generate_invoice(
+            self.partner_1,
+            self.partner_2,
+            move_type='out_invoice',
+            invoice_line_ids=[
+                {
+                    'product_id': self.product_a.id,
+                    'quantity': 2.0,
+                    'product_uom_id': self.env.ref('uom.product_uom_dozen').id,
+                    'price_unit': 990.0,
+                    'tax_ids': [(6, 0, self.tax_21.ids)],
+                },
+            ],
+        )
+        invoice.action_post()
+
+    def test_export_import_invoice(self):
```

Comment:
this is weird... I was expecting:
def test_export_invoice_...(self):
def test_export_refund_...(self):

def test_import_invoice_...(self):
def test_import_refund_...(self):

...each one being a dedicated xml file

Also, plz make `test_files` a subfolder of `tests` and rename the files according to the tests like:
test_fr_export_invoice_....xml

Also, I don't really like all of these PDFs files. What about putting only the xmls?